code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -257,6 +264,13 @@ obj/machinery/chem_dispenser/RefreshParts()
 			dispensable_reagents -= list("goldschlager","patron","watermelonjuice","berryjuice")
 			hackedcheck = 0
 			return
+	if(istype(B, /obj/item/weapon/wrench))

```

Comment:
This check isn't necessary, it inherits this wrench check from the parent object type